.github/workflows/linting.yml

```diff
@@ -358,7 +358,7 @@ jobs:
       - name: Monorepo pnpm install
         run: pnpm install
       - name: Run lint-style
-        run: pnpm lint-style .
+        run: pnpm lint-style . -f verbose
```

Comment:
This is only for testing for this PR, right?

Reply:
I was considering leaving it since a successful run showed no output, but have now removed it in favor of a simple `echo`, since it dumps all the checked files AFTER the errors instead of before.